content/en/observability_pipelines/guide/custom-metrics-governance.md

```diff
@@ -118,13 +117,12 @@ transforms:
     type: remap
     inputs:
       - dd_agent.metrics
-    source: |2+
-
-           # list of tags to keep
-           tags_blocklist = ["tag 1", "tag 2", "tag 3", "tag 4"]
+    source: |
+      # list of tags to keep
+      tags_blocklist = ["tag 1", "tag 2", "tag 3", "tag 4"]
 
-           # filter and drop any tags that don't match the tags in tags_to_keep
-           .tags = [filter(.tags) -> |_index, value| { !includes(tags_blocklist, value)}
+      # filter and drop any tags that don't match the tags in tags_to_keep
```

Comment:
Should `tags_to_keep` here be `tags_allowlist`?

You should also change the `tags_blocklist` local above to `tags_allowlist`